src/vim9cmds.c

```diff
@@ -174,6 +174,31 @@ compile_unlet(
     return ret;
 }
 
+// TOOD: Probably belongs in vim9compile next to lookup_local.
+//	 Or maybe by itself.
+    static int
+lookup_static(char_u *name, size_t len, lvar_T *lvar, cctx_T *cctx)
```

Comment:
Can you use cctx_class_member_idx() here?

Reply:
Yes